src/main/java/com/frameworkium/config/drivers/ElectronImpl.java

```diff
@@ -11,12 +11,19 @@
 import java.util.Map;
 
 import static com.frameworkium.config.SystemProperty.APP_PATH;
-import static com.frameworkium.config.SystemProperty.GRID_URL;
 
 public class ElectronImpl extends DriverType {
 
     private static URL remoteURL;
 
+    static {
+        try {
+            remoteURL = new URL("http://localhost:9515");
+        } catch (MalformedURLException e) {
+            e.printStackTrace();
```

Comment:
Could we log this as debug (instead of print), throw the exception or just remove this line if we don't care about it failing?